File: NBA_Scrape/model/match_score.py
# -*- coding: utf-8 -*-
# 比赛小节得分(比赛编号、小节、主队得分、客队得分)

class MatchScore(object):

	# 限制Match_score的属性
	__slots__ = (
		'__game_id','__section','__home_point','__guest_point'
	)
	def __init__(self):
		self.__game_id = None
		self.__section = -1
		self.__home_point = -1
		self.__guest_point = -1

	# 比赛编号
	@property
	def game_id(self):
		return self.__game_id
	
	@game_id.setter
	def game_id(self, value):
		self.__game_id = value
	
	# 小节
	@property
	def section(self):
		return self.__section
	
	@section.setter
	def section(self, value):
		self.__section = value
	
	# 主队得分
	@property
	def home_point(self):
		return self.__home_point
		
	@home_point.setter
	def home_point(self, value):
		self.__home_point = value
		
	# 客队得分
	@property
	def guest_point(self):
		return self.__guest_point
		
	@guest_point.setter
	def guest_point(self, value):
		self.__guest_point = value

File: NBA_Server/src/main/java/entity/MatchScore.java
package entity;

/**
 * 比赛小节记分实体类
 * 
 * created by JaneLDQ on 2015年5月17日 下午12:50:09
 */
public class MatchScore {

	/**
	 * 比赛编号
	 */
	private String game_id;
	
	/**
	 * 小节
	 */
	private Integer section;
	
	/**
	 * 主队得分
	 */
	private Integer home_point;
	
	/**
	 * 客队得分
	 */
	private Integer guest_point;
	
	/**
	 * 无参构造函数
	 */
	public MatchScore(){}

	public String getGame_id() {
		return game_id;
	}

	public void setGame_id(String game_id) {
		this.game_id = game_id;
	}

	public Integer getSection() {
		return section;
	}

	public void setSection(Integer section) {
		this.section = section;
	}

	public Integer getHome_point() {
		return home_point;
	}

	public void setHome_point(Integer home_point) {
		this.home_point = home_point;
	}

	public Integer getGuest_point() {
		return guest_point;
	}

	public void setGuest_point(Integer guest_point) {
		this.guest_point = guest_point;
	}
	
	
}
